sig-testing/charter.md

```diff
@@ -11,6 +11,10 @@ away project toil. We focus on creating and running tools and infrastructure
 that make it easier for the community to write and run tests, and to
 contribute, analyze and act upon test results.
 
+We define the overall testing standards, strategies and style guidelines for
+the project, although, each SIG is responsible of the implementation and
+interpretation of these guidelines.
```

Comment:
Slight wording tweak:

```suggestion
the project, although, each SIG is responsible for the implementation and
interpretation of these guidelines in the context of their work.
```

I'm still a bit uneasy about the "interpretation of these guidelines": if the guidelines need interpretation, then they were not clear enough. I think what this says is "need to decide to which extend they want to follow those guidelines".

So how about this:
```suggestion
the project. Each SIG is responsible for determining how those
guidelines apply to their work and implementing those parts
which make sense.
```

Reply:
I agree.